test/jdk/tools/jlink/plugins/CDSPluginTest.java

```diff
@@ -69,8 +72,14 @@ public static void main(String[] args) throws Throwable {
             subDir = "lib" + sep;
         }
         subDir += "server" + sep;
-        helper.checkImage(image, module, null, null,
-                          new String[] { subDir + "classes.jsa", subDir + "classes_nocoops.jsa" });
+
+        if (Platform.isAArch64() || Platform.isX64()) {
+            helper.checkImage(image, module, null, null,
+                      new String[] { subDir + "classes.jsa", subDir + "classes_nocoops.jsa" });
+        } else {
+            helper.checkImage(image, module, null, null,
+                      new String[] { subDir + "classes.jsa" });
+        }
```

Comment:
The above change checking 64-bit platform should not be needed.    On an unknown platform, `isDefaultCDSArchiveSupported` should return false.  It only returns true on 64-bit supported platforms.   Can you verfiy?

Reply:
I just try it on LoongArch64 and mips64el.

The `Platform.isDefaultCDSArchiveSupported()` returns true.
So the above checking can not be removed.

Also, I just forgot add `import jtreg.SkippedException`.